eng/Version.Details.xml

```diff
@@ -145,14 +145,14 @@
       <SourceBuild RepoName="linker" ManagedOnly="true" />
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.3.0-3.22305.4" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.3.0-2.22307.7" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@marcpopMSFT @mmitche any idea why this is getting a downgrade❔ Did the toolset get special-cased in dotnet/sdk for example❔

Reply:
seems odd - was there a downgrade in the SDK that got merged and then flowed here. That's the only way it would happen  AFAIK

Reply:
This was an intentional downgrade in SDK. Roslyn folks told me to merge it offline as they had had some issues in their codeflow so this is expected.